content/pages/education/opt-out-information-sharing/opt-out-form-0993.md

```diff
@@ -10,7 +10,7 @@ description: Use your VA education benefits to pay for college or training progr
     <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
       <li><a href="/">Home</a></li>
       <li><a href="/education/">Education</a></li>
-      <li><a aria-current="page" href="/education/opt-out-information-sharing/">Opt out of sharing your information with schools</a></li>
+      <li><a aria-current="page" href="/education/opt-out-information-sharing/opt-out-form-0993">Opt out of sharing your information with schools</a></li>
```

Comment:
Another reason to move the introduction page into the react app, is that it will not be blocked in production. This link has been changed temporarily to the form, as the static page is blocked in production.